ir/inst_other.go

```diff
@@ -157,7 +157,8 @@ type InstPhi struct {
 	// extra.
 
 	// Type of result produced by the instruction.
-	Typ types.Type // type of incoming value
+	Typ           types.Type // type of incoming value
+	FastMathFlags []enum.FastMathFlag
```

Comment:
```suggestion
		// (optional) Fast math flags.
		FastMathFlags []enum.FastMathFlag
```

For consistency with https://github.com/llir/llvm/blob/0d19cd0df110ed9c8fbcec42e3a0239456b15e2c/ir/inst_other.go#L93

```go
type InstFCmp struct {
...
	// Type of result produced by the instruction.
	Typ types.Type // boolean or boolean vector
	// (optional) Fast math flags.
	FastMathFlags []enum.FastMathFlag
```